docker/images/github-cli/Dockerfile

```
@@ -2,13 +2,15 @@ ARG base_image=bitnami/minideb:latest
 
 FROM ${base_image}
 
-RUN apt-get update -qq && apt-get install -y ca-certificates git
+RUN apt-get update -qq && apt-get install -y ca-certificates git pip
```

Not sure pip is available. You could also install wget and [download the binary](https://github.com/mikefarah/yq#wget).

Something like:

```suggestion
RUN apt-get update -qq && apt-get install -y ca-certificates git wget
RUN wget https://github.com/mikefarah/yq/releases/download/v4.25.3/yq_linux_amd64.tar.gz -O - |  tar xz && mv yq_linux_amd64 /usr/bin/yq
```

---

I'm getting this when I run locally:
```
> docker run --rm -it bitnami/minideb:latest

root@4c807f6aadfa:/# apt-get update -qq && apt-get install -y ca-certificates git pip
Reading package lists... Done
Building dependency tree... Done
E: Unable to locate package pip
```

---

i thought about it then choose against it as it will not auto-update?

---

weird, I just build it.

---

just ran what you did on your machine and it works on mine, could it be because I'm on M1 chip

---

You can use `https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64.tar.gz` to get the latest release, if that helps.

---

I agree it'd be preferable to use the package though.